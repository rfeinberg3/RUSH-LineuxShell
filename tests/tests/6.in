path /autograder/source/bin
p3.sh & p5.sh & p2.sh
exit
