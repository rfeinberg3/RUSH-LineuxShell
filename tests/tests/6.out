rush> rush> elephant
cat
panther
rush> 